test/unittests/OpModel/TTNN/Conversion/TestConversion.cpp

```diff
@@ -200,35 +200,45 @@ INSTANTIATE_TEST_SUITE_P(
         std::make_tuple(TensorMemoryLayout::WidthSharded,
                         llvm::SmallVector<int64_t>{32, 56 * 32},
                         llvm::SmallVector<int64_t>{1, 56},
-                        CoreRangeSet{
-                            CoreRange(CoreCoord(0, 0), CoreCoord(7, 6))}),
-        std::make_tuple(TensorMemoryLayout::WidthSharded,
-                        llvm::SmallVector<int64_t>{32, 13 * 32},
-                        llvm::SmallVector<int64_t>{1, 13},
-                        CoreRangeSet{std::set<CoreRange>{
-                            CoreRange(CoreCoord(0, 0), CoreCoord(7, 0)),
-                            CoreRange(CoreCoord(0, 1), CoreCoord(4, 1))}}),
+                        ::tt::tt_metal::CoreRangeSet{::tt::tt_metal::CoreRange(
+                            tt::tt_metal::CoreCoord(0, 0),
+                            tt::tt_metal::CoreCoord(7, 6))}),
+        std::make_tuple(
+            TensorMemoryLayout::WidthSharded,
+            llvm::SmallVector<int64_t>{32, 13 * 32},
+            llvm::SmallVector<int64_t>{1, 13},
+            ::tt::tt_metal::CoreRangeSet{std::set<tt::tt_metal::CoreRange>{
+                ::tt::tt_metal::CoreRange(tt::tt_metal::CoreCoord(0, 0),
+                                          tt::tt_metal::CoreCoord(7, 0)),
+                ::tt::tt_metal::CoreRange(tt::tt_metal::CoreCoord(0, 1),
+                                          tt::tt_metal::CoreCoord(4, 1))}}),
```

Comment:
:warning: **clang\-diagnostic\-error** :warning:
no member named `` tt_metal `` in namespace `` mlir::tt ``; did you mean `` ::tt::tt_metal ``?
```suggestion
                                          ::tt::tt_metal::CoreCoord(4, 1))}}),
```